package.json

```
@@ -6,23 +6,24 @@
     },
     "version": "12.1.0",
     "source": "src/index.ts",
-    "main": "dist/index.js",
-    "module": "dist/index.modern.js",
-    "unpkg": "dist/index.umd.js",
+    "main": "dist/commonjs/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/types/index.d.ts",
     "publishConfig": {
         "access": "public"
     },
     "files": [
         "dist/**/*"
     ],
     "scripts": {
-        "build": "rollup --config && yarn createTsDec",
-        "createTsDec": "tsc --emitDeclarationOnly true --declaration true --declarationDir dist/ --noEmit false",
+        "build": "yarn build:esm && yarn build:commonjs",
+        "build:esm": "tsc",
+        "build:commonjs": "tsc --outDir dist/commonjs --module CommonJS",
```

you might want to target ES2018 here, to be sure it will run happily in node 10 and above
```suggestion
        "build:commonjs": "tsc --outDir dist/commonjs --module CommonJS --target ES2018",
```

---

Does that clash with [our recommendations](https://github.com/guardian/recommendations/blob/master/npm-packages.md#package-content)?

---

it [won't](https://github.com/guardian/recommendations/pull/55/files#diff-105c2556445e6cf4212095164b2c536c0f3197ec34872e7368dde59a6b6fde36R27)